scripts/contractInteraction/contract_interaction.py

```diff
@@ -939,4 +943,60 @@ def determineFundsAtRisk():
     print(len(borrowedPositions))
     print('total height of affected loans: ', sum/1e18)
     print('total potential borrowed: ', possible/1e18)
-    print('could have been stolen: ', (possible - sum)/1e18)
\ No newline at end of file
+    print('could have been stolen: ', (possible - sum)/1e18)
+
+def createProposalSIP0014():
+    # action
+    target = contracts['SOV']
+    signature = "name()"
+    data = "0x"
+    description = "SIP-0014: Strategic Investment, Details: https://github.com/DistributedCollective/SIPS/blob/18f7eec97d3e280f45601cf879a7cda9985c522d/SIP-0014.md, sha256: 2dc16befc5c2733bfbb8fcd81e4b3726904183f2a26618696f31059523466499"
+
+    governor = Contract.from_abi("GovernorAlpha", address=contracts['GovernorOwner'], abi=GovernorAlpha.abi, owner=acct)
+
+    print('Governor Address:    '+governor.address)
+    print('Target:              '+str([target]))
+    print('Values:              '+str([0]))
+    print('Signature:           '+str([signature]))
+    print('Data:                '+str([data]))
+    print('Description:         '+str(description))
+    print('======================================')
+
+    # # create proposal
+    # governor.propose(
+    #     [target],
+    #     [0],
+    #     [signature],
+    #     [data],
+    #     description)
+
+def addInvestorToBlacklist():
+    # we need to process CSOV->SOV exchnage manually,
+    # investor address should be added to blacklist in VestingRegistry
+    tokenOwner = "0x75F7d09110631FE60a804642003bE00C8Bcd26b7"
+
+    vestingRegistry = Contract.from_abi("VestingRegistry", address=contracts['VestingRegistry'], abi=VestingRegistry.abi, owner=acct)
+    data = vestingRegistry.setBlacklistFlag.encode_input(tokenOwner, True)
+    print(data)
+
+    # multisig = Contract.from_abi("MultiSig", address=contracts['multisig'], abi=MultiSigWallet.abi, owner=acct)
+    # tx = multisig.submitTransaction(SOVtoken.address,0,data)
```

Comment:
Shouldn't the target address be the `vestingRegistry.address`?

Reply:
no, we need to transfer from GovernorVaultOwner